circle.yml

```diff
@@ -1,7 +1,7 @@
 # Warning: This file is automatically synced from https://github.com/ipfs/ci-sync so if you want to change it, please change it there and ask someone to sync all repositories.
 machine:
   node:
-    version: stable
```

Comment:
Please use LTS

Reply:
If we're supposed to be standardising on Jenkins, do you think it might be better to remove the travis & circle setup instead?

Reply:
✅  from me. cc @VictorBjelkholm

Reply:
Yeah, code coverage doesn't have a solution as of now. See https://github.com/ipfs/wg-qa-testing-dx/issues/23#issuecomment-386346304

Once we have that in place, we can start removing Travis and Circle from everywhere.

Reply:
Hmm, I changed it to be node version 8 and it's broken the circle build.  Which is odd because the travis build passes with node 8.  Circle complains about leveldown being compiled for a later version of node - do we cache the dependencies between Circle builds?

Reply:
@achingbrain the caching could be a bit funky in CircleCI somtimes, I've retried the job with cache reset this time. Running here: https://circleci.com/gh/ipfs/js-ipfs-mfs/34

Reply:
Nice one, thanks - that appears to have fixed it.